PrivacyPolicy.md

```diff
@@ -0,0 +1,42 @@
+# Information collected and transmitted by Git Extensions
+
+We would like your permission to collect a little information about how you use Git Extensions. This will help us to make the application better.
+
+First, a reminder: Git Extensions is provided "as is", without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement.
+
+With that out of the way, here's a breakdown of all the information we may collect via Application Insights.
+
+### Application-Level
+Includes:
+* Exception information (could, in rare cases, contain paths on your computer)
+  * Version number (e.g. 2.0.x.x)
+* Build type (whether the application is an official release build or not)
+* Selected layout settings (such as visibility of the left panel, commit info position etc)
+* Change of selected layout settings
+* Git version (e.g. 2.19.0)
+* SSH client (e.g. OpenSSH or PuTTY)
+
+
+### Operating System-Level
+Includes:
+    * Version (e.g. Windows 10.0.17763.0)
+    * Machine Name (e.g. MyFastPC)
```

Comment:
Can this be removed, too?

Reply:
Unless you're explicitly setting the "RoleInstance" property in an initializer, AppInsights core library will set it to the hostname. It's your choice, but if you do that, I'd include it in the disclosures.

I find OS version quite useful to record so you know what API's you can safely use.

Reply:
@mstv could you elaborate why you are concerned about the hostname?

For me it seems more effort in removing it since it is captured by AppInsights libraries. 
(To a degree) I trust these libraries to make reasonable decisions about collecting a minimal subset of data.

Reply:
I meant only the machine name. It might contain *personal or identifiable information*, e.g. the owner's name.

Reply:
Using Oren's tip, it appeared relatively painless to override the hostname capture.